server/app.js

```diff
@@ -3,10 +3,19 @@ const express = require("express");
 const { join } = require("path");
 const cookieParser = require("cookie-parser");
 const logger = require("morgan");
+const mongoose = require("mongoose");
 
 const indexRouter = require("./routes/index");
 const pingRouter = require("./routes/ping");
 
+// Connect to the database
+const mongoDB = "mongodb://127.0.0.1/mail-sender-dev";
```

Comment:
This should variable ideally should be stored in a config file, and then the config file should be referencing an environment variable